config/environments/production.rb

```diff
@@ -1,3 +1,5 @@
+ENV['HOST_URL'] = 'sport-portal-dev.herokuapp.com'
+
 # Configure airbrake, error reporting to Errbit
```

Comment:
What is the default option for the action mailer in production?

Reply:
none for now. I will copy the dev settings and open an issue that another mail address should get acquired. What do you think?

Reply:
No, please revert the change below